infrastructure/helm/quantumserverless/README.md

```diff
@@ -59,6 +67,12 @@ Install from specific values file
 kubectl patch svc -n ray kuberay-apiserver-service --type json  --patch '[{"op" : "replace" ,"path" : "/spec/selector" ,"value" : {"app.kubernetes.io/component": "kuberay-apiserver"}}]'
 ```
 
+(temporary) Patch the kuberay-apiserver deployment
```

Comment:
Sorry that I couldn't have time to review this pull request properly 🙏 

My only question for the future would be how we can remove this patch? I assume that is something related with ray like the patch that we have above, doesn't it, @psschwei , @akihikokuroda ?

Reply:
That's right. The fix(es) has already been merged upstream, so once there's a new release cut we can bump our deps and drop the patch(es)

Reply:
My question was more related to what actions do we need to apply to remove the patch (or not need it)? Like for example with the first one, that we need to open an issue in ray to fix the `app.kubernetes.io/component` annotation. I assume that the problem in this case is that we can't add as sidecar in the `kuberay-api ` the `gatekeeper` image? To have it under control and move these things as soon as I can.

Reply:
For the sidecar patch, the sidecar setting is already in the values.yaml so when kuberay has a new release and we upgrade, we can just remove the patch script and update the README.  For the service, we can just update the README when we get the new kuberay release.